### PR TITLE
Replace-text-in-list-paragraph

### PR DESCRIPTION
**Pull request by DharanyaSakthivel-SF4210**
Description:
Added sample for replace text in list paragraph

### CODE REVIEW

File: HTML-conversions/Replace-text-in-list-paragraph/.NET/Replace-text-in-list-paragraph/Replace-text-in-list-paragraph.csproj [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  <ItemGroup>
+    <PackageReference Include="Syncfusion.DocIO.Net.Core" Version="*" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="Data\File.html">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+    <None Update="Data\Input.docx">
```
Comment (MohanaselvamJothi) [inline HTML-conversions/Replace-text-in-list-paragraph/.NET/Replace-text-in-list-paragraph/Replace-text-in-list-paragraph.csproj:17]: Check whether it is need?

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: HTML-conversions/Replace-text-in-list-paragraph/.NET/Replace-text-in-list-paragraph/Program.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.IO;
+
+namespace Replace_text_in_list_paragraph
```
Comment (MohanaselvamJothi) [inline HTML-conversions/Replace-text-in-list-paragraph/.NET/Replace-text-in-list-paragraph/Program.cs:5]: Change sample name as Replace text in list  paragraph with HTML

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: HTML-conversions/Replace-text-in-list-paragraph/.NET/Replace-text-in-list-paragraph/Program.cs (hunk 2 above)
Comment (DharanyaSakthivel-SF4210) [reply HTML-conversions/Replace-text-in-list-paragraph/.NET/Replace-text-in-list-paragraph/Program.cs]: modified

File: HTML-conversions/Replace-text-in-list-paragraph/.NET/Replace-text-in-list-paragraph/Replace-text-in-list-paragraph.csproj (hunk 1 above)
Comment (DharanyaSakthivel-SF4210) [reply HTML-conversions/Replace-text-in-list-paragraph/.NET/Replace-text-in-list-paragraph/Replace-text-in-list-paragraph.csproj]: Removed the gitkeep file.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**